specs/typography.md

```diff
@@ -0,0 +1,177 @@
+
+*Base size*
+
+While the default styling could vary from browser to browser, most seem to adopt the same default styling for compatibility with existing pages. The default base font size is **16px**.
+
+*Heading size*
+
+[Heading element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements) font sizes are calculated from the base size value as follows:
+
+| Element    | Relative | Size   |
+|------------|----------|--------|
+| ```<h1>``` | 2em      | 32px   |
+| ```<h2>``` | 1.5em    | 24px   |
+| ```<h3>``` | 1.17em   | 18.7px |
+| ```<h4>``` | 1em      | 16px   |
+| ```<h5>``` | 0.83em   | 13.2px |
+| ```<h6>``` | 0.67em   | 10.7px |
+
+Any of these elements may be used with FAST-DNA styles applied.
+
+### Use Cases
+
+- Defining page hierarchy through one or more sequences of ```<h1>```, ```<h2>```, ```<h3>```, etc.
+- Associating a ```<label>``` with an input field
+- Styling text as secondary or metatext
+- Adding a caption to an image or figure
+
+### Features
+
+- An extensible type ramp based on a default font size
+- A set of semantically named styles based on the underlying type ramp
+- A default mapping for style to standard elements
+
+### Risks and Challenges
+
+It's possible that styles only (not components) will be harder to implement consistently. We may want to consider simple custom elements that consistently apply the same underlying styles as well as any necessary aria tags.
+
+### Prior Art/Examples
+
+- FAST-DNA (React): [Heading](https://explore.fast.design/components/heading), [Subeading](https://explore.fast.design/components/subheading), [Paragraph](https://explore.fast.design/components/paragraph), [Label](https://explore.fast.design/components/label), [Caption](https://explore.fast.design/components/caption), [Metatext](https://explore.fast.design/components/metatext), [Typography](https://explore.fast.design/components/typography)
+- [Material Design](https://material.io/design/typography/the-type-system.html)
+- [Bootstrap](https://getbootstrap.com/docs/4.4/content/typography)
+- [Ant Design](https://ant.design/components/typography)
+- [Semantic UI](https://semantic-ui.com/elements/header.html)
+- [Fluent UI](https://fluentsite.z22.web.core.windows.net/components/header/definition)
+
+Most other component frameworks either maintain the base font size or set it smaller to 14px. They also typically maintain heading element sizes based on `em` or `rem`, though some use individual specific values. Typically heading sizes are close to the browser default, though Material Design goes a lot larger at the top.
+
+---
+
+## Design
+
+The type styles published are consistently named so they are easy to identify and consume. A clear separation is made between the semantic naming and the raw type ramp.
+
+Default styles are applied to standard elements such that headings and other base typography pick up the adaptive styling without needing to add class names.
```

Comment:
This can have interop issues with components and will often lead to significant overrides of base elements within components - historically we've avoided doing this.

Reply:
I would like to understand the principle that's driving this part of the proposal and how it relates to our goals in FAST-DNA.  As mentioned above, traditionally we have avoided this and I think there is often good reason for doing so.

Reply:
One note on this that I forgot about - because of the isolation provided by web components we won't really have a leaking problem here. It likely isn't as big of an issue as it has been previously